From fd0acfbe79444687c73dea182c2d1a5fa1c77324 Mon Sep 17 00:00:00 2001
From: Michal Vasilek <michal@vasilek.cz>
Date: Sat, 15 Jan 2022 00:07:31 +0100
Subject: [PATCH] build: remove positional i18n.merge_file arguments

otherwise building with meson 0.61+ fails:

    ERROR: Function does not take positional arguments.
---
 data/meson.build | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/data/meson.build b/data/meson.build
index d32638a8..2dc1798b 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -20,7 +20,6 @@ resources = gnome.compile_resources ('org.gnome.Boxes',
 desktop_conf = configuration_data ()
 desktop_conf.set ('icon', application_id)
 desktop = i18n.merge_file (
-  'desktop',
   input: configure_file (
     input: files ('org.gnome.Boxes.desktop.in'),
     output: 'org.gnome.Boxes.desktop.in',
@@ -65,7 +64,7 @@ configure_file (
 
 appdata_conf = configuration_data()
 appdata_conf.set('appid', application_id)
-appdata_file = i18n.merge_file ('appdata-file',
+appdata_file = i18n.merge_file (
   input: configure_file (
     input: files ('org.gnome.Boxes.appdata.xml.in'),
     output: 'org.gnome.Boxes.appdata.xml.in',
-- 
2.34.1

